hudi-common/src/main/java/org/apache/hudi/metadata/HoodieTableMetadataUtil.java

```diff
@@ -753,7 +752,7 @@ public static HoodieData<HoodieRecord> convertMetadataToColumnStatsRecords(Hoodi
           String partitionPath = deleteFileInfoPair.getLeft();
           String filePath = deleteFileInfoPair.getRight();
 
-          if (filePath.endsWith(HoodieFileFormat.PARQUET.getFileExtension()) || ExternalFilePathUtil.isExternallyCreatedFile(filePath)) {
+          if (ExternalFilePathUtil.isExternallyCreatedFile(filePath)) {
```

Comment:
This change is causing `TestExternalPathHandling.testFlow` to fail as the stats are excluded for replaced parquet file. May I know what's the intention behind this change?

I guess the intention might be that any files cleaned (data files as well as log files) should have the entries removed from col stats right? But, then should we just change the condition to be something like: `if (filePath.endsWith(HoodieFileFormat.PARQUET.getFileExtension()) || FSUtils.isLogFile(filePath) || ExternalFilePathUtil.isExternallyCreatedFile(filePath))`

Reply:
I have removed the entire if case here. we don't it anymore. 
can you check latest commit. I have fixed it. 
it was an issue w/ the clean.